### ISSUE TITLE
[BUG] Example store is broken

### ISSUE THREAD

**Issue by IceflowRE** [refinedev/refine#3168]
### Describe the bug

1. Dependency failed to resolve

 ```
Could not resolve dependency:
npm ERR! peer react@"^16.8 || ^17.0" from @radix-ui/react-dropdown-menu@0.1.6
npm ERR! node_modules/@radix-ui/react-dropdown-menu
npm ERR!   @radix-ui/react-dropdown-menu@"^0.1.6" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: react@17.0.2
npm ERR! node_modules/react
npm ERR!   peer react@"^16.8 || ^17.0" from @radix-ui/react-dropdown-menu@0.1.6
npm ERR!   node_modules/@radix-ui/react-dropdown-menu
npm ERR!     @radix-ui/react-dropdown-menu@"^0.1.6" from the root project
```

2.

```
Error: Failed to parse src "RFNtshirt-develop-lightblue-model-1662488307284.png" on `next/image`, if using relative image it must start with a leading slash "/" or be an absolute URL (http:// or https://)
    at defaultLoader (/Users/iceflower/refine/examples/store/node_modules/next/dist/shared/lib/image-loader.js:28:23)
    at eval (webpack-internal:///./node_modules/next/dist/client/image.js:94:39)
    at Array.map (<anonymous>)
    at generateImgAttrs (webpack-internal:///./node_modules/next/dist/client/image.js:94:24)
    at eval (webpack-internal:///./node_modules/next/dist/client/image.js:552:27)
    at renderWithHooks (/Users/iceflower/refine/examples/store/node_modules/react-dom/cjs/react-dom-server.browser.development.js:5658:16)
    at renderForwardRef (/Users/iceflower/refine/examples/store/node_modules/react-dom/cjs/react-dom-server.browser.development.js:5842:18)
    at renderElement (/Users/iceflower/refine/examples/store/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6005:11)
    at renderNodeDestructiveImpl (/Users/iceflower/refine/examples/store/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6104:11)
    at renderNodeDestructive (/Users/iceflower/refine/examples/store/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6076:14)
error - Error: Failed to parse src "RFNtshirt-develop-lightblue-model-1662488307284.png" on `next/image`, if using relative image it must start with a leading slash "/" or be an absolute URL (http:// or https://)
    at defaultLoader (/Users/iceflower/refine/examples/store/node_modules/next/dist/shared/lib/image-loader.js:28:23)
    at eval (webpack-internal:///./node_modules/next/dist/client/image.js:94:39)
    at Array.map (<anonymous>)
    at generateImgAttrs (webpack-internal:///./node_modules/next/dist/client/image.js:94:24)
    at eval (webpack-internal:///./node_modules/next/dist/client/image.js:552:27)
    at renderWithHooks (/Users/iceflower/refine/examples/store/node_modules/react-dom/cjs/react-dom-server.browser.development.js:5658:16)
    at renderForwardRef (/Users/iceflower/refine/examples/store/node_modules/react-dom/cjs/react-dom-server.browser.development.js:5842:18)
    at renderElement (/Users/iceflower/refine/examples/store/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6005:11)
    at renderNodeDestructiveImpl (/Users/iceflower/refine/examples/store/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6104:11)
    at renderNodeDestructive (/Users/iceflower/refine/examples/store/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6076:14) {
  page: '/'
}
```

### Steps To Reproduce

Follow the given guide in the example readme.

### Expected behavior

-

### Desktop

MacOS 13.0.1

**Comment by omeraplak** [comment]
Hey @IceflowRE ,
Wow, you are so fast! We'll fix it soon 🎯

**Comment by omeraplak** [comment]
Hey @IceflowRE ,
We've fixed issues with #3171 🛒
